### PR TITLE
Custom navtech servers

### PR DESCRIPTION
**Pull request by ro-savage**
This PR adds custom NavTech severs to NavPay.

- [x] Add ability to save, delete and read NavTech servers
- [x] Add settings page for NavTech servers
- [x] Add input on send page, if no NavTech servers found
- [x] Add verification of servers when adding custom servers
- [x] Test real-world private payments through custom servers
- [x] Check styling on iphone, android, and desktop
- [ ] Update https://navtechservers.com to use URL instead of IP

### CODE REVIEW

**Comment by ro-savage** [comment]
### Android
Android: https://s3-ap-southeast-2.amazonaws.com/random-navcore-dev-stuff/navpay-custom-navtech.apk
- [x] (Android) Add custom servers from settings screen
- [x] (Android) Add incorrect custom server from settings screen
- [x] (Android) Delete custom servers from settings screen
- [x] (Android) With no custom servers. Attempt private payment from send screen. Add Server.
- [x] (Android) Make actual private payment

### Web
Web & iOS: https://navpay-custom-navtech.now.sh 
- [x] (Web) Add custom servers from settings screen
- [x] (Web) Add incorrect custom server from settings screen
- [x] (Web) Delete custom servers from settings screen
- [x] (Web) With no custom servers. Attempt private payment from send screen. Add Server.
- [x] (Web) Make actual private payment

File: package.json [hunk 1]
```diff
@@ -3,10 +3,7 @@
   "description": "A Secure NavCoin Wallet",
   "author": "NavPay",
   "version": "4.1.0",
-  "keywords": [
-    "navocin",
-    "wallet"
-  ],
+  "keywords": ["navcoin", "wallet", "nav", "NavPay"],
```
Comment (matt-auckland) [inline package.json]: why do we do a lowercase navcoin but mixed case NavPay ?

Comment (ro-savage) [reply package.json]: No reason :)

File: www/views/tab-send.html [hunk 2]
```diff
@@ -35,18 +35,35 @@
...
         <div class="send-private-toggle">
-          <ion-toggle ng-model="formData.privatePayment" toggle-class="toggle-balanced">
+          <ion-toggle ng-click="togglePrivatePayment()" toggle-class="toggle-balanced">
             <span class="toggle-label" translate>Private Payment</span>
           </ion-toggle>
         </div>
+        <div ng-show="showAddNavTech" class="addNavTechIPContainer">
+          <div ng-hide="navTechAddressSuccess">
+            <p>No NavTech servers found. Please add a NavTech server to make private payments.</p>
+            <p>See <a href="https://navtechservers.com/">navtechservers.com</a> for more details.</p>
```
Comment (matt-auckland) [inline www/views/tab-send.html]: This hyperlink doesn't work on android

Comment (ro-savage) [reply www/views/tab-send.html]: Any idea how to make links work on Android?

File: package.json (hunk 1 above)
Comment (zanuka) [reply package.json]: all the keywords are belong to us

**Comment by ro-savage** [comment]
Hyperlinks updated by @matt-auckland. Tested by @ro-savage.

Approved.

**Pull request merged**